style.css

```diff
@@ -0,0 +1,53 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;1,100;1,200;1,300;1,500;1,600;1,700&display=swap');
+
+.main_container {
+  background-color: #e5e5e5;
+  max-width: 375px;
+  height: 718px;
```

Comment:
- [ ] OPTIONAL
Good job for updating the height of the app. However, defining a height will cause your app not to fit the height of some mobile devices, as can be seen on the attached screenshot below of _Apple XR_. I'll recommend that you remove the device height

**Heigh unfitting screen:**
![image](https://user-images.githubusercontent.com/12745474/170529909-f238bd79-f08e-40c0-b3e2-b556e5d0af52.png)